### DISCUSSION TITLE
INC module labels

### DISCUSSION THREAD

**Discussion by yiliu30** [intel/neural-compressor#1582] [Show and tell]
### Module Tags

- Better categorize the PRs and issues.
- Labels list:`INC3.X`, `Auto-Tune`, `PyTorch`,`Onnx Runtime`,`Tensorflow`

https://github.com/intel/neural-compressor/issues/labels

**Comment by yiliu30** [comment]
https://docs.github.com/en/search-github/searching-on-github/searching-issues-and-pull-requests#search-by-label
